client/src/sass/include/_variables.scss

```diff
@@ -116,7 +116,7 @@ $variables: (
   --embedBigPlayBackgroundColor: var(--embedBigPlayBackgroundColor)
 );
 
-@function var($variable) {
+@function pvar($variable) {
```

Comment:
I'm not sure to understand this change.

Reply:
css variables are also called with 'var()' in pure css.